proposals/jvm-static-annotation-in-interface-companion.md

```diff
@@ -0,0 +1,36 @@
+# @JvmStatic for interface companion members
+
+* **Type**: Design proposal
+* **Author**: Mike Bogdanov
+* **Status**: Implemented in Kotlin 1.3
+* **Discussion**: [KEEP-150](https://github.com/Kotlin/KEEP/issues/150)
+
+## Summary
+
+Allow to use '@JvmStatic' annotation on interface companion members similar to class companion ones.
```

Comment:
Consider using backticks (\`) instead of single quotation marks, so that such text is rendered `verbatim` with appropriate style for code.